src/java/picard/sam/RevertSam.java

```diff
@@ -78,18 +82,43 @@
             "information, which can be used to produce an unmapped BAM (uBAM) from a previously aligned BAM. It is also capable of " +
             "restoring the original quality scores of a BAM file that has already undergone base quality score recalibration (BQSR) if the" +
             "original qualities were retained." +
-            "<h4>Usage example:</h4>" +
+            "<h4>Usage example 1:</h4>" +
             "<pre>" +
             "java -jar picard.jar RevertSam \\<br />" +
             "     I=input.bam \\<br />" +
             "     O=reverted.bam" +
             "</pre>" +
+            "Output format is BAM by default, or SAM if the input path ends with '.sam'." +
+            "<h4>Usage example 2:</h4>" +
```

Comment:
I agree, specifying in the title of the example what distinguishes/characterizes it is very helpful.